x-pack/plugins/maps/public/actions/data_request_actions.ts

```diff
@@ -299,7 +299,6 @@ function onDataLoadError(
     dispatch(cleanTooltipStateForLayer(layerId));
     dispatch({
       type: LAYER_DATA_LOAD_ERROR,
-      data: null,
```

Comment:
No point sending null in action. Reducer can just set data to null when resolving.